R/pkg/inst/tests/testthat/test_sparkSQL.R

```diff
@@ -668,6 +668,15 @@ test_that("collect() returns a data.frame", {
   df <- createDataFrame(list(list(1, 2)), schema = c("name", "name"))
   ldf <- collect(df)
   expect_equal(names(ldf), c("name", "name"))
+  createOrReplaceTempView(df, "dfView")
```

Comment:
since this is the `collect DataFrame` test, perhaps you want to move these to the `sql` test or the `createOrReplaceTempView` test?